stackslib/src/main.rs

```diff
@@ -215,13 +216,17 @@ fn main() {
 
         let block_path = &argv[2];
         let block_data = fs::read(block_path).expect(&format!("Failed to open {}", block_path));
+        let epoch_id = parse_input_epoch(3);
```

Comment:
It seems like none of these commands or their args are documented in `stacks-node --help`. This should be fixed in a separate PR